tensorflow/BUILD

```diff
@@ -1665,18 +1657,13 @@ gen_api_init_files(
     output_dir = "_api/v1/",
     output_files = TENSORFLOW_API_INIT_FILES_V1,
     output_package = "tensorflow._api.v1",
+    packages_to_ignore = ["tensorflow.python.framework.test_ops"],
     root_file_name = "v1.py",
-    root_init_template = "$(location api_template_v1.__init__.py)",
+    root_init_template = "api_template_v1.__init__.py",
 )
 
-gen_api_init_files(
+generate_apis(
     name = "tf_python_api_gen_v2",
-    srcs = [
-        "api_template.__init__.py",
-        "compat_template.__init__.py",
-        "compat_template_v1.__init__.py",
-    ],
-    api_packages_file_name = "api_packages.txt",
```

Comment:
I think we need to add support for `api_packages_file_name` so that the wheel test introduced in https://github.com/tensorflow/tensorflow/commit/2289daf62fa657f080c4e3dd0b8c3b4fada52d64 still works.